MON-19264 - Documentation - Depreciate the ip label datametrie monitoring connector (#2432)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Warning

> This plugin pack has been marked as deprecated, since the Datamétrie online 

Reviewer: ```suggestion
This connector is deprecated, as the Datamétrie online 
```

> This connector is deprecated, as the Datamétrie online 
> service is not available any more.

Reviewer 2: ```suggestion
> This connector is deprecated, as the Datamétrie online 
> service is not available any more.
```
